packages/entrykit/src/quarry/transports/methods/estimateUserOperationGas.ts

```diff
@@ -0,0 +1,20 @@
+import { BundlerRpcSchema } from "viem";
+import { formatUserOperationRequest } from "viem/account-abstraction";
+import { getRpcMethod } from "../common";
+
+// TODO: revisit after demo (don't hardcode gas)
+
+type rpcMethod = getRpcMethod<BundlerRpcSchema, "eth_estimateUserOperationGas">;
+
+export async function estimateUserOperationGas(
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  _params: rpcMethod["Parameters"],
+): Promise<rpcMethod["ReturnType"]> {
+  return formatUserOperationRequest({
+    callGasLimit: 20_000_000n,
+    preVerificationGas: 200_000n,
+    verificationGasLimit: 2_000_000n,
+    paymasterVerificationGasLimit: 200_000n,
+    paymasterPostOpGasLimit: 200_000n,
+  });
+}
```

Comment:
wonder if for now we should do the rpc call and accept the additional round trip rather than hardcoding by default

Reply:
iirc an rpc call is done automatically when this isn't defined here, and there were issues estimating gas when lazily creating the smart account (both passkey and session account)

I can remove for now to see if it works, and if so, revisit this when we get back to passkey smart accounts